utilities/question_query_utils.py

```diff
@@ -59,7 +59,7 @@ def parse_tag(text: str) -> Optional[str]:
         return None
     tag_val = match.group(1)
     tag_pat = _tag_pat.replace(r"\s", " ")
-    tag_idx = tag_pat.lower().find(tag_val.lower())
+    tag_idx = tag_pat.lower().find(rf"\b{tag_val.lower()}\b") + 2
```

Comment:
what's the `+ 2` part about, please?

Reply:
it's to offset the index so that it matches the start of the tag in the pattern
"\b" is 2 characters

Reply:
huh? `\b` is an escape sequence that matches zero characters at the [word boundary](https://www.regular-expressions.info/wordboundaries.html), the lenght of the regex input shouldn't matter, only the match, no? or what am I missing here?

Reply:
oh, it's a non-regex `find` inside a string that represents a regex for future match 🤦 

I'm not sure why we have a list of tags in the format of a string that represents regex, but if it's not possible to use an actual list of tags like `['tag1', 'tag2']`, then I guess the code is fine...